{
  Throwable t=exceptionEvent.getCause();
  if (t instanceof IOException) {
    LOG.debug(t,t);
    ctx.getChannel().close();
  }
 else {
    this.exceptionCaught(t);
  }
}
